change-notes/1.23/analysis-python.md

```diff
@@ -20,8 +32,21 @@
 |----------------------------|------------------------|------------|
 | Unreachable code | Fewer false positives | Analysis now accounts for uses of `contextlib.suppress` to suppress exceptions. |
 | `__iter__` method returns a non-iterator | Better alert message | Alert now highlights which class is expected to be an iterator. |
-
+| Explicit return in __init__ method | Fewer false positives | Instances where the `__init__` method returns the value of a call to a procedure are no longer flagged. |
```

Comment:
Please make the following change to this column.
```suggestion
| Explicit return in __init__ method | Fewer false positive results | Instances where the `__init__` method returns the value of a call to a procedure are no longer flagged. |
```

Reply:
Done.